### PR TITLE
Render HTML in announcements' messages as expected

### LINKED ISSUE #2641: [dev.icinga.com #13523] Links in announcements are not rendered as html link
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13523

**Created by tobiasvdk on 2016-12-11 20:34:12 +00:00**

Assignee: _(none)_
Status: _New_ 
Target Version: _(none)_
Last Update: _2016-12-11 20:34:12 +00:00 (in Redmine)_

---

Links in announcements are not rendered as html link - they are not "clickable".

### PR DESCRIPTION
**Pull request by Al2Klimov**
fixes #2641

### CODE REVIEW

**Comment by lippserd** [comment]
We'll consider this for the 2.6.0 release.

**Comment by nilmerg** [comment]
@lippserd Please review.

File: library/Icinga/Web/Helper/HtmlPurifier.php [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+     *
+     * @param   string          $html
+     * @param   array|Closure   $config     Additional configuration
+     *
+     * @return  string
+     */
+    public static function process($html, $config = null)
+    {
+        $purifier = new static($config);
+        return $purifier->purify($html);
```
Comment (lippserd) [inline library/Icinga/Web/Helper/HtmlPurifier.php]: Please add a newline.

**Comment by nilmerg** [comment]
@lippserd Done.

**Comment by lippserd** [comment]
Thanks guys 👍

**Pull request merged**